packages/back-end/src/app/controllers/aws-healthomics/workflow/read-private-workflow.lambda.ts

```diff
@@ -0,0 +1,86 @@
+import { ResourceNotFoundException } from '@aws-sdk/client-omics';
+import { GetWorkflowCommandInput } from '@aws-sdk/client-omics/dist-types/commands/GetWorkflowCommand';
+import { Laboratory } from '@easy-genomics/shared-lib/src/app/types/easy-genomics/laboratory';
+import { buildErrorResponse, buildResponse } from '@easy-genomics/shared-lib/src/app/utils/common';
+import {
+  LaboratoryNotFoundError,
+  OmicsWorkflowNotFoundError,
+  RequiredIdNotFoundError,
+  UnauthorizedAccessError,
+} from '@easy-genomics/shared-lib/src/app/utils/HttpError';
+import { APIGatewayProxyResult, APIGatewayProxyWithCognitoAuthorizerEvent, Handler } from 'aws-lambda';
+import { LaboratoryService } from '@BE/services/easy-genomics/laboratory-service';
+import { OmicsService } from '@BE/services/omics-service';
+import {
+  validateLaboratoryManagerAccess,
+  validateLaboratoryTechnicianAccess,
+  validateOrganizationAdminAccess,
+} from '@BE/utils/auth-utils';
+
+const laboratoryService = new LaboratoryService();
+const omicsService = new OmicsService();
+
+/**
+ * This GET /aws-healthomics/workflow/read-private-workflow/{:id}?workflowId={workflowId}
```

Comment:
Just noticed this typo comment which should read: `/aws-healthomics/workflow/read-private-workflow/{:id}?laboratoryId={LaboratoryId}`